tests/support/case.py

```diff
@@ -788,6 +790,17 @@ class ModuleCase(TestCase, SaltClientTestCaseMixin):
     Execute a module function
     '''
 
+    masterminion = None
+
+    def __init__(self, *args, **kwargs):
+        super(ModuleCase, self).__init__(*args, **kwargs)
+        if self.masterminion is None:
+            opts = self.get_config(
+                self._salt_client_config_file_name_,
+                from_scratch=True,
+            )
+            ModuleCase.masterminion = salt.minion.MasterMinion(opts)
```

Comment:
Why not self?

Reply:
I know we only instantiate the class object once, but we should either do something in `setUpClass` or... something.
Because we should delete the object once we're done with the test module class.

We'll also have to instantiate it several times...

I guess we can leave as is, and see if we get  object leaks...

Reply:
I *wanted* to just make it a classlevel, to avoid creating it umptymillion times, but the `_salt_client_config_filename_` came from self. It might not really be important, though.

Reply:
Yeah, I understand why you did it.
But we've had issues with not releasing objects before hence my worries.
But lets wait and see.